package.json

```diff
@@ -45,57 +48,58 @@
     "license:check": "npm run license:list -- --failOn `cat .licensebanned | grep '^[^#;]' | awk 'BEGIN { ORS=\"\" } { print p$0\";\"; } END { print \n }'`",
     "dep:check": "npx ncu -e 2",
     "dep:update": "npx ncu -u",
-    "release": "standard-version --releaseCommitMessageFormat 'chore(release): {{currentTag}} [skip ci]'"
+    "release": "standard-version --releaseCommitMessageFormat 'chore(release): {{currentTag}} [skip ci]'",
+    "snapshot": "standard-version --no-verify --skip.changelog --prerelease snapshot --releaseCommitMessageFormat 'chore(snapshot): {{currentTag}}'"
   },
   "dependencies": {
-    "@hapi/boom": "9.1.4",
+    "@hapi/boom": "10.0.0",
     "@hapi/good": "9.0.1",
-    "@hapi/hapi": "20.2.1",
+    "@hapi/hapi": "20.2.2",
     "@hapi/inert": "6.0.5",
     "@hapi/vision": "6.1.0",
-    "@mojaloop/central-services-error-handling": "11.3.0",
-    "@mojaloop/central-services-health": "13.0.0",
-    "@mojaloop/central-services-logger": "10.6.2",
-    "@mojaloop/central-services-metrics": "11.0.0",
-    "@mojaloop/central-services-shared": "16.0.0",
-    "@mojaloop/event-sdk": "10.7.1",
-    "ajv": "8.10.0",
+    "@mojaloop/central-services-error-handling": "12.0.4",
+    "@mojaloop/central-services-health": "14.0.1",
+    "@mojaloop/central-services-logger": "11.0.1",
+    "@mojaloop/central-services-metrics": "12.0.5",
+    "@mojaloop/central-services-shared": "17.0.2",
+    "@mojaloop/event-sdk": "11.0.2",
+    "ajv": "8.11.0",
     "ajv-keywords": "5.1.0",
     "blipp": "4.0.2",
-    "commander": "9.0.0",
+    "commander": "9.3.0",
     "docdash": "1.2.0",
-    "hapi-swagger": "14.2.5",
+    "hapi-swagger": "14.5.5",
     "joi": "17.6.0",
     "mustache": "4.2.0",
     "parse-strings-in-object": "2.0.0",
     "rc": "1.2.8"
   },
   "devDependencies": {
     "@mojaloop/ml-testing-toolkit-shared-lib": "12.2.0",
-    "@types/jest": "27.4.1",
-    "eslint": "7.32.0",
-    "eslint-config-standard": "16.0.3",
-    "eslint-plugin-import": "2.24.2",
+    "@types/jest": "28.1.4",
+    "eslint": "8.0.1",
+    "eslint-config-standard": "17.0.0",
+    "eslint-plugin-import": "2.26.0",
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-promise": "5.1.0",
-    "eslint-plugin-standard": "4.1.0",
+    "eslint-plugin-promise": "6.0.0",
+    "eslint-plugin-standard": "5.0.0",
     "get-port": "5.1.1",
-    "jest": "27.5.1",
-    "jest-junit": "13.0.0",
+    "jest": "28.1.2",
+    "jest-junit": "14.0.0",
     "jsdoc": "3.6.10",
     "license-checker": "25.0.1",
-    "npm-audit-resolver": "2.3.1",
-    "npm-check-updates": "12.5.2",
+    "npm-audit-resolver": "3.0.0-7",
+    "npm-check-updates": "15.2.6",
     "pre-commit": "1.2.2",
-    "sinon": "13.0.1",
-    "standard": "16.0.4",
-    "standard-version": "9.3.2"
+    "sinon": "14.0.0",
+    "standard": "17.0.0",
+    "standard-version": "9.5.0"
```

Comment:
We are missing the `standard-version` config from here: https://github.com/mojaloop/central-ledger/blob/master/package.json#L150

Don't forget to add the `replace` dev dependency.